examples/inabox.host.html

```diff
@@ -59,68 +59,6 @@
       width="300" height="1000">
   </iframe>
 
-  <script>
-    initInaboxHost(window);
-    // Enable amp-inabox APIs for all iframes
-    Array.prototype.push.apply(
-        window.ampInaboxIframes, document.querySelectorAll('iframe'));
-
-    /**
-     * A sample bootstrap script that is to be run in ads tag.
-     *
-     * @param {!Window} win
-     */
-    function initInaboxHost(win) {
-      const hostScriptUrl = 'http://localhost:8000/dist/amp-inabox-host.js';
-      preloadScript(win, hostScriptUrl);
-      // Optional: preload a4a-v0.js as well.
-
-      // Keep all iframes that potentially can render AMP content in this array
-      win.ampInaboxIframes = win.ampInaboxIframes || [];
-      // Keep all amp-inabox post-messages in this array.
-      win.ampInaboxPendingMessages = win.ampInaboxPendingMessages || [];
-
-      let hostScriptRequested = false;
-
-      const listener = event => {
-        if (!isInaboxMessage(event.data)) {
-          return;
-        }
-        win.ampInaboxPendingMessages.push(event);
-
-        // Load and execute host script once.
-        if (hostScriptRequested) {
-          return;
-        }
-        hostScriptRequested = true;
-        // Load inabox-host.js when the 1st inabox message is received.
-        loadScript(win, hostScriptUrl, () => {
-          win.removeEventListener('message', listener);
-          console.log('a4a-host.js loaded.');
-        });
-      };
-      win.addEventListener('message', listener);
-      console.log('A4A host bootstrapped.');
-    }
-
-    function isInaboxMessage(message) {
-      return typeof message === 'string' && message.indexOf('amp-') == 0;
-    }
-
-    function preloadScript(win, url) {
-      const link = win.document.createElement('link');
-      link.rel = 'preload';
-      link.href = url;
-      link.as = 'script';
-      win.document.head.appendChild(link);
-    }
-
-    function loadScript(win, url, callback) {
-      const script = win.document.createElement('script');
-      script.src = url;
-      script.onload = callback;
-      win.document.body.appendChild(script);
-    }
-  </script>
+  <script src="./inabox.host-bootstrap.js"></script>
```

Comment:
did you check in the new file?
Also, how about calling it: `./inabox-tag-integration.js`